device-types/FS/S5860-20SQ.yaml

```diff
@@ -0,0 +1,73 @@
+---
+manufacturer: FS
+model: S5860-20SQ
+slug: s5860-20sq
+part_number: s5860-20sq
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+console-ports:
```

Comment:
Please add `weight` information if you have it, according to the datasheet above it would be:
```
weight:
  - value: 10.14
    unit: lb
```

Reply:
Fixed on #cb670f548df0b69e618c51b3b2c9341d8c1d228c